### PR TITLE
Fix 'Send' button behavior for the case there is one account

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

File: src/modals/send-modal/send-modal.form.tsx [hunk 1]
```diff
@@ -31,6 +31,6 @@ export const sendModalValidationSchema: SchemaOf<SendModalFormValues> = object()
 
       return false;
     }),
-  ownAccount: object().shape({}).required(),
+  ownAccount: object().shape({}).optional(),
```
Comment (unixvb) [inline src/modals/send-modal/send-modal.form.tsx:34]: maybe `ownAccount` filed should be required when `transferBetweenOwnAccounts === true`?  (the same what you did with `receiverPublicKeyHash`)

Comment (keshan3262) [reply src/modals/send-modal/send-modal.form.tsx:34]: Thanks, I've already tried f**king with those `yup` typings

**Review by andrewkryshtal — APPROVED** [review]

File: src/modals/send-modal/send-modal.form.tsx (hunk 1 above)
Comment (unixvb) [reply src/modals/send-modal/send-modal.form.tsx:34]: you could just use object literal and `@ts-ignore` in this case, to ignore types

Comment (keshan3262) [reply src/modals/send-modal/send-modal.form.tsx:34]: Okay, I thought that was strictly forbidden

**Review by unixvb — APPROVED** [review]

**Pull request merged**